server/config/cron-tasks.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
 				const records = await getPluginService(strapi, 'actionService').find({
 					filters: {
 						executeAt: {
-							$lte: new Date(),
+							$lte: (new Date()).getTime(),
```

Comment:
This is equivalent to Date.now() which makes is the preferred method of retrieving the unix epoch.

```suggestion
							$lte: Date.now(),
```